src/main/scala/io/iohk/ethereum/consensus/ethash/difficulty/ConfiguredDifficultyCalculator.scala

```diff
@@ -0,0 +1,36 @@
+package io.iohk.ethereum.consensus.ethash.difficulty
+
+import io.iohk.ethereum.consensus.difficulty.DifficultyCalculator
+import io.iohk.ethereum.domain.BlockHeader
+
+class ConfiguredDifficultyCalculator(powTargetTime: Long) extends DifficultyCalculator {
+
+  val DifficultyBoundDivision: Int = 2048
+  val FrontierTimestampDiffLimit: Int = -99
+  val MinimumDifficulty: BigInt = 131072
+
+  /**
+    * The LowerBoundExpectedRatio (l for abbreviation below) divides the timestamp diff into ranges:
+    *   [0, l)   => c = 1, difficulty increases
+    *   [l, 2*l) => c = 0. difficulty stays the same
+    *   ...
+    *   [l*i, l*(i+1) ) => c = 1-i, difficulty decreases
+    *
+    *  example:
+    *  powTargetTime := 60 seconds
+    *  l := 30 seconds
+    *   [0, 0.5 min)         => difficulty increases
+    *   [0.5 min, 1.5 min)   => difficulty stays the same (the average should be powTargetTime)
+    *   [1.5 min, +infinity) => difficulty decreases
+    */
+  val LowerBoundExpectedRatio: Long = powTargetTime / 2
```

Comment:
Why `powTargetTime / 2` here? Doesn't that give the ranges:
```
[0, time/2) = [0, 22.5)       => difficulty increases
[time/2, time) = [22.5, 45)   => difficulty stays the same
[time, 3/2 time) = [45, 67.5) => difficulty decreases
```
Meaning that the block time would be around: 37 seconds?

On the ETC calculator they are using `3/2 * powTargetTime` (LowerBoundExpectedRatio=10, powTargetTime=15) as their LowerBoundExpectedRatio, shouldn't we use the the same?